contract/src/test/java/io/neow3j/contract/GasTokenTest.java

```diff
@@ -14,19 +14,13 @@
 
 public class GasTokenTest {
 
-    @Rule
-    public WireMockRule wireMockRule = new WireMockRule(options().dynamicPort());
-
     private Neow3j neow;
 
     private final static String GASTOKEN_SCRIPTHASH = "70e2301955bf1e74cbb31d18c2f96972abadb328";
 
     @Before
     public void setUp() {
-        // Configuring WireMock to use default host and the dynamic port set in WireMockRule.
-        int port = this.wireMockRule.port();
-        WireMock.configureFor(port);
-        neow = Neow3j.build(new HttpService("http://127.0.0.1:" + port));
+        neow = Neow3j.build(new HttpService("http://127.0.0.1:8080"));
```

Comment:
the port is not supposed to be dynamic?

Reply:
The service is not actually used by all of the tests in that class. I should make that more clear.